website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -0,0 +1,200 @@
+
+- `Hds::Form::TextInput` add support for `datetime-local` type. [(#1494)](https://github.com/hashicorp/design-system/pull/1494) Thanks [@natmegs](https://github.com/natmegs)! 
+
+- `Hds::SideNav` - updated layout styling for the `SideNav::List::Title` element. [(#1500)](https://github.com/hashicorp/design-system/pull/1500) Thanks [@didoo](https://github.com/didoo)! 
+
+- Upgrade `ember-style-modifier` to `3.0.1`. [(#1520)](https://github.com/hashicorp/design-system/pull/1520) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- 🔄 Updated dependencies:
+    - @hashicorp/design-system-tokens@1.7.0
+    - @hashicorp/ember-flight-icons@3.0.8
+
+## 2.8.0
+
+**Month Day, Year**
+
+- `Hds::Card` ⛔️ **Breaking change** updated default value of @overflow argument to "visible" to address an area of consumer confusion and better support the most common use cases. 
+[(#1492)](https://github.com/hashicorp/design-system/pull/1492) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 
+    - Technically, this is a breaking change as it will require consumers relying upon the previous hidden default value to now manually set the value. The result of not setting the a hidden value can cause square edges of some images to "stick out" and overlap the rounded corners of the Card itself. We considered this to be a fairly minor, low-impact issue however which would not affect functionality or usability.
+
+- `Hds::Form::TextInput` add loading state on "search" type. [(#1452)](https://github.com/hashicorp/design-system/pull/1492) Thanks [@alex-ju](https://github.com/alex-ju)! 
+
+- `Hds::Form::MaskedInput` add component. [(#1468)](https://github.com/hashicorp/design-system/pull/1492) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `Hds::Accordion component` add component [(#1423)](https://github.com/hashicorp/design-system/pull/1423) Thanks [@KristinBradley](https://github.com/KristinLBradley)!
+
+- `Hds::PageHeader` set position to 'relative'. [(#1466)](https://github.com/hashicorp/design-system/pull/1466) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `Hds::Textarea` fix border and text color for readonly state. [(#1470)](https://github.com/hashicorp/design-system/pull/1470) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `Hds::Modal` prevent onClose callback function invocation when isDismissDisabled is true. [(#1456)](https://github.com/hashicorp/design-system/pull/145) Thanks [@alex-ju](https://github.com/alex-ju)!
+
+- `font-weight` of the `button` mixin explicitly to `regualr` instead of relying on inheritance (components using this mixin: `Button`, `Dropdown::ToggleButton` and soon `Accordion`) - No visual difference expected. [(#1469)](https://github.com/hashicorp/design-system/pull/1469) Thanks [@didoo](https://github.com/didoo)!
+
+- 🔄 Updated dependencies:
+    - @hashicorp/design-system-tokens@1.6.0
+    - @hashicorp/ember-flight-icons@3.0.7
```

Comment:
```suggestion
**July 13, 2023**

- `Hds::Card` - updated default value of @overflow argument to "visible" to address an area of consumer confusion and better support the most common use cases. [#1492](https://github.com/hashicorp/design-system/pull/1492) Thanks [@KristinLBradley](https://github.com/KristinLBradley)! 

    - Technically, this is a breaking change as it will require consumers relying upon the previous hidden default value to now manually set the value. The result of not setting the a hidden value can cause square edges of some images to "stick out" and overlap the rounded corners of the Card itself. We considered this to be a fairly minor, low-impact issue however which would not affect functionality or usability.

- `Hds::Form::TextInput` - added loading state on "search" type. [#1452](https://github.com/hashicorp/design-system/pull/1492) Thanks [@alex-ju](https://github.com/alex-ju)! 

- `Hds::Form::MaskedInput` - added component. [#1468](https://github.com/hashicorp/design-system/pull/1492) Thanks [@alex-ju](https://github.com/alex-ju)!

- `Hds::Accordion` - added component [#1423](https://github.com/hashicorp/design-system/pull/1423) Thanks [@KristinBradley](https://github.com/KristinLBradley)!

- `Hds::PageHeader` - set position to 'relative'. [#1466](https://github.com/hashicorp/design-system/pull/1466) Thanks [@alex-ju](https://github.com/alex-ju)!

- `Hds::Textarea` - fixed border and text color for readonly state. [#1470](https://github.com/hashicorp/design-system/pull/1470) Thanks [@alex-ju](https://github.com/alex-ju)!

- `Hds::Modal` - prevented onClose callback function invocation when isDismissDisabled is true. [#1456](https://github.com/hashicorp/design-system/pull/145) Thanks [@alex-ju](https://github.com/alex-ju)!

- Set the `font-weight` of the `button` mixin explicitly to `regular` instead of relying on inheritance (components using this mixin: `Button`, `Dropdown::ToggleButton` and soon `Accordion`) - No visual difference expected. [#1469](https://github.com/hashicorp/design-system/pull/1469) Thanks [@didoo](https://github.com/didoo)!

- 🔄 Updated dependencies:
    - @hashicorp/design-system-tokens@1.6.0
    - @hashicorp/ember-flight-icons@3.0.7
```